### PR TITLE
Add Skinnyh's story

### PR DESCRIPTION
**Pull request by skinnyh**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1631)

### CODE REVIEW

File: Chinese/Skinnyh/Skinnyh.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+大白 [Click here](../Baymax/Baymax.md)
+Mayday [Click here](../mayDay/mayDay.md)
```
Comment (casey-collab) [inline Chinese/Skinnyh/Skinnyh.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: Chinese/Skinnyh/Skinnyh.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+大白 [Click here](../Baymax/Baymax.md)
```
Comment (casey-collab) [inline Chinese/Skinnyh/Skinnyh.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by skinnyh** [comment]
Great!

**Comment by skinnyh** [comment]
Please review this, @casey-collab

File: Chinese/Skinnyh/Skinnyh.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Mayday [Click here](../mayDay/mayDay.md)
```
Comment (casey-collab) [inline Chinese/Skinnyh/Skinnyh.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by skinnyh** [comment]
Please review this, @casey-collab

File: Chinese/Skinnyh/Skinnyh.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Mayday 是五月天
```
Comment (casey-collab) [inline Chinese/Skinnyh/Skinnyh.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@skinnyh, if a reader starts at `language.md` and keeps clicking links, they will not be able to reach your addition. Is your change meant to be a part of the existing story? (I can't tell because I don't speak Chinese.) If so, then you should find the place within the existing story where your addition belongs and add a link. If not, then it's fine that you made this pull request to test how pull requests work, but I only want to merge changes that are a part of the story, so I'm going to close it.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**